code/modules/vending/_vending.dm

```diff
@@ -342,6 +343,15 @@ GLOBAL_LIST_EMPTY(vending_machines_to_restock)
 	if(light_mask && !(machine_stat & BROKEN) && powered())
 		. += emissive_appearance(icon, light_mask, src)
 
+/obj/machinery/vending/examine(mob/user)
```

Comment:
Don't forget to add an examine for having credits to collect since you have a tip for it